service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadTranslationsSubmittedControllerTest.java

```diff
@@ -0,0 +1,284 @@
+            eq(ITEM_TRANSLATED_NOTIFICATION_TEMPLATE),
+            eq(LOCAL_AUTHORITY_1_INBOX), eqJson(NOTIFICATION_PARAMETERS),
+            eq(NOTIFICATION_REFERENCE)
+        ));
+    }
+
+    @Test
+    void shouldSendOrdersByPostWhenOrderTranslated() {
+        CaseData caseData = caseData();
+        CallbackRequest request = CallbackRequest.builder()
+            .caseDetails(asCaseDetails(caseData))
+            .build();
+
+        postSubmittedEvent(request);
+
+        checkUntil(() -> {
+            verify(sendLetterApi, timeout(ASYNC_METHOD_CALL_TIMEOUT).times(2)).sendLetter(eq(SERVICE_AUTH_TOKEN),
+                printRequest.capture());
+            verify(coreCaseDataService).updateCase(eq(CASE_ID), caseDataDelta.capture());
+        });
+
+        LetterWithPdfsRequest expectedPrintRequest1 = printRequest(
+            CASE_ID, ORDER, COVERSHEET_REPRESENTATIVE_BINARY, ORDER_BINARY
+        );
+
+        LetterWithPdfsRequest expectedPrintRequest2 = printRequest(
+            CASE_ID, ORDER, COVERSHEET_RESPONDENT_BINARY, ORDER_BINARY
+        );
+
+        assertThat(printRequest.getAllValues()).usingRecursiveComparison()
+            .isEqualTo(List.of(expectedPrintRequest1, expectedPrintRequest2));
+
+        List<Element<SentDocuments>> documentsSent = mapper.convertValue(
+            caseDataDelta.getValue().get("documentsSentToParties"), new TypeReference<>() {}
+        );
+
+        SentDocument expectedRepresentativeDocument = documentSent(
+            REPRESENTATIVE_POST.getValue(), COVERSHEET_REPRESENTATIVE, ORDER_DOCUMENT, LETTER_1_ID, now()
+        );
+
+        SentDocument expectedRespondentDocument = documentSent(
+            RESPONDENT_NOT_REPRESENTED.getParty(), COVERSHEET_RESPONDENT, ORDER_DOCUMENT, LETTER_2_ID, now()
+        );
+
+        assertThat(documentsSent.get(0).getValue().getDocumentsSentToParty())
+            .extracting(Element::getValue)
+            .containsExactly(expectedRepresentativeDocument);
+
+        assertThat(documentsSent.get(1).getValue().getDocumentsSentToParty())
+            .extracting(Element::getValue)
+            .containsExactly(expectedRespondentDocument);
+    }
+
+    @Test
+    void shouldNotifyRepresentativesServedDigitallyWhenOrderTranslated() {
+        CaseData caseData = caseData();
+        CallbackRequest request = CallbackRequest.builder()
+            .caseDetails(asCaseDetails(caseData))
+            .build();
+
+        postSubmittedEvent(request);
+
+        checkUntil(() -> verify(notificationClient, timeout(ASYNC_METHOD_CALL_TIMEOUT)).sendEmail(
+            eq(ITEM_TRANSLATED_NOTIFICATION_TEMPLATE),
+            eq(REPRESENTATIVE_DIGITAL.getValue().getEmail()), eqJson(NOTIFICATION_PARAMETERS),
+            eq(NOTIFICATION_REFERENCE)
+        ));
+    }
+
+    @Test
+    void shouldNotifyRepresentativesServedByEmailWhenOrderTranslated() {
+        CaseData caseData = caseData();
```

Comment:
can just inline case data into the the post submitted, don't need to explicitly convert to callback request

Reply:
same for all the other methods

Reply:
done